experiments/ClimaEarth/run_amip.jl

```diff
@@ -126,29 +126,56 @@ add_extra_diagnostics!(config_dict)
     plot_diagnostics,
 ) = get_coupler_args(config_dict)
 
+#=
+### I/O Directory Setup `setup_output_dirs` returns `dir_paths.output =
```

Comment:
I'd rather keep this comment in the function in src/Utilities.jl and not duplicate it in the driver as well

Reply:
The driver doubles as tutorial, so I left it here because I think it's very important to explain where one will find the output.

(I'd be fine with stop treating the driver as a tutorial)